### PR TITLE
Add EGM96 geoid grid.

### PR DESCRIPTION
**Pull request by kbevers**
Adding the `egm96_15.gtx` grid so that the proj-datumgrids package provides the necessary resources needed for the tests in PROJ.

### CODE REVIEW

File: README.DATUMGRID [hunk 1]
```diff
@@ -161,6 +161,14 @@ NAD83 to NAD83.
...
 1-dimensional transformation grids for vertical datum shifts.
 
+### Worldwide: EGM96 geoid model
+
+*Source*: [NGA](http://earth-info.nga.mil/GandG/wgs84/gravitymod/egm96/egm96.html)  
+*Format*: GTX  
+*License*: Public domain  
+
+15 minute worldwide geoid undulation grid that transforms WGS84 ellipsoidal heights to physical heights.
+
```
Comment (sebastic) [inline README.DATUMGRID]: Should mark the file as included in:
```
*Package:* proj-datumgrid
```

File: README.DATUMGRID [hunk 2]
```diff
@@ -161,6 +161,14 @@ NAD83 to NAD83.
 
 1-dimensional transformation grids for vertical datum shifts.
 
+### Worldwide: EGM96 geoid model
+
+*Source*: [NGA](http://earth-info.nga.mil/GandG/wgs84/gravitymod/egm96/egm96.html)  
+*Format*: GTX  
+*License*: Public domain
```
Comment (sebastic) [inline README.DATUMGRID]: For consistency capitalize "Domain" as used by the other grids.

**Review by sebastic — CHANGES_REQUESTED** [review]
Looks good, only two minor changes needed.

**Comment by kbevers** [comment]
Thanks. I also forgot to write the name of the grid.

I'll add a `-europe` and `-northamerica` packages during the next couple of hours. I'll update the main README with a description of the structure and move the relevant grid info to the regional packages. I think that makes it more clear what's what across the different archives. Do you have any input in that regard?

**Comment by sebastic** [comment]
I see some value in keeping all the grids documented in `README.DATUMGRID`. The main README directs users to the region specific package which contain the grids they need. Moving the grid documentation to the region subdirectory requires users to navigate the directory structure.

Perhaps this is not much of an issue if structure in the main README is clear enough that grids for those countries can be found in their respective subdirectories.

**Review by sebastic — APPROVED** [review]

**Comment by kbevers** [comment]
>Perhaps this is not much of an issue if structure in the main README is clear enough that grids for those countries can be found in their respective subdirectories.

Yeah, I was thinking of adding a section something along the lines of:

## Regional grid archives

In addition to the core proj-datumgrid archive regional grid collections also exist. These include grids that are not essential for the functionality of PROJ but still useful for PROJ-users in the specific regions.

### Europe
Includes grids for:
* Denmark
* The Faroe Islands
* ...

See README.EUROPE for grid details.
### North America
Includes grids for 
* Greenland
* ...

See README.NORTHAMERICA for grid details.

### Oceanica
Includes grids for
* Australia
* ...

See README.OCEANIA for grid details.

**Comment by sebastic** [comment]
That would also reduce the need to mark the package for each grid, just documenting the package for the each region should suffice.

Where to find the region README files /or hyperlinking them is probably good. And/Or mentioning the directory where the region grids live.

This discussion should probably be moved to #7 where the context is more appropriate.

**Pull request merged**